video/qt_decoder.cpp

```diff
@@ -498,7 +498,27 @@ const Graphics::Surface *QuickTimeDecoder::VideoTrackHandler::decodeNextFrame()
 	if (endOfTrack())
 		return 0;
 
+	// Playback rate logic, if rate != -1 then seek to our rate adjusted time!
+	if (_decoder->getRate() != 1 && _decoder->getRate() != -1) {
+		uint32 time = _decoder->getTime();
```

Comment:
I do not see an absolute need to add this variable. Moreover, the name is problematic because it is clashing with the standard function `time()`. Please write `seek(_decoder->getTime())`